src/ui/units/wizard/components/Dialogs/DialogColor/DialogColorGradient/DialogColorGradient.tsx

```diff
@@ -30,6 +33,11 @@ export const AUTO = 'auto';
 const MANUAL = 'manual';
 const TOOLTIP_DELAY_CLOSING = 250;
 
+const NULLS_OPTIONS = [
+    {value: 'ignore', content: i18n('wizard', 'label_painting-ignore')},
+    {value: 'as-0', content: i18n('wizard', 'label_painting-as-0')},
```

Comment:
You can use GradientNullModes values instead of strings

Reply:
fixed